tutorials/gSlack/index.md

```diff
@@ -0,0 +1,172 @@
+# gSlack
+
+This tutorial will explain how to get notified by [Slack](https://slack.com/) of various events that originate within [Google Cloud Platform](https://cloud.google.com/). `Google Compute Engine` instances may start and stop, new `Google AppEngine` verions might be deployed and `Google Cloud Storage` bucket may be created or deleted. People would like to be notified of these and others via their `Slack` account.
+
+`gSlack` allows you to install a lightweight serverless solution into your
+own Google Cloud Platform project/s and easily configure the alerts you would like to receive.
+
+## Objectives
+In this tutorial you will learn how to:
+- Enable `Slack` integration.
+- Create `Google PubSub` a topic.
+- Create `Google Stackdriver` log exports that write into the `PubSub` topic.
+- Configure permissions for the log exports to be written into the `PubSub`.
+- Configure and deploy a `Google CloudFunctions`(**beta**) serverless backend code to call the `Slack` API.
+- Configure the alerts to notify of.
+
+## Before you begin
+This tutorial runs best when using a `*nix` operating system.
+
+If you don't have access to a `*nix` machine you may choose to use `Google Cloud Shell` using this [quickstart](https://cloud.google.com/shell/docs/quickstart).
```

Comment:
Add comma:
If you don't have access to a \`*nix\` machine`,` you may choose to use...